pkg/cmd/dev/test.go

```diff
@@ -38,16 +39,45 @@ const (
 	vModuleFlag      = "vmodule"
 )
 
+var testTargetMapping = map[string]string{
```

Comment:
For these too, I don't see much value or possible usage. I'm rarely ever running package wide tests, and even when I am, it's usually targeted depending on the area of the code I'm editing and/or dependant packages; ala `dev test pkg/spanconfig/...`. It's possible already to run all tests through `dev test pkg/...`, which mirrors the `go test ...` selector syntax, and for CRDB would take forever). The random assortment of packages you'd get by bazel package size seems a lot less useful IMO. 

I'd prefer removing these entirely, but if not, using flags instead of named targets here. So `dev test --{small,medium,large,enormous}`. In fact, if I did only care about running tests by size, being able to combine them with recursive package selectors + filtering using these flags feels like it would compose better. So something like `dev test pkg/... --small`.